src/player/index.ts

```diff
@@ -133,6 +132,8 @@ export class CodeTourComment implements Comment {
 
     this.body = new MarkdownString(body);
     this.body.isTrusted = true;
+    this.body.supportHtml = true;
```

Comment:
I'm super curious - do these props work like we expected from our meeting earlier this week? Or what's the before and after behavior for setting them?

Reply:
improves the view and it now reads the icons better so I guess they are working as expected. I don't think they improve screen reader experience as much though, but setting them is the right thing to do imo

Reply:
ok sounds reasonable, thanks for the details!